### PR TITLE
Video to audio converter

### LINKED ISSUE #429: Video to Audio converter
# Aim

- A simple automation script to convert any type of video to audio.

#GSSOC'21

### PR DESCRIPTION
**Pull request by smriti1313**
# Description

A python script to convert video to audio.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@avinashkranjan There is confusion here. Issue #321 was assigned to @kshittijagrawal for which he made PR #481.

But another issue #429 with the same task was assigned to @smriti1313 for which this PR was made. 

Both have implemented with the same library with different approaches. (P.S: @smriti1313 code with minor bug fix which I did locally will work for all file types but @kshittijagrawal code is restricted to some file types only. Kindly suggest the next step

File: Video_to_audio/video_to_audio.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+arg_parser.add_argument("-v", "--videoFile", required=True,
+                    help="Video Filename with the complete path.")
+arg_parser.add_argument('-p', "--path", required=True,
+                    help="Audio Filename with absolute or relative path")
+
+args = vars(arg_parser.parse_args())
+
+try:
+    Video_clip = pyeditor.VideoFileClip(r"{}".format(args['videoFile']))
+    video_clip.audio.write_audiofile(r"{}".format(args['path']))
```
Comment (kaustubhgupta) [inline Video_to_audio/video_to_audio.py:15]:
```suggestion
    Video_clip.audio.write_audiofile(r"{}".format(args['path'])))
```

**Review by kaustubhgupta — COMMENTED** [review]
Hi @smriti1313, 
As there was a confusion of Scripts, the final decision is that both: your and @kshittijagrawal contribution will be considered. 

Now the important change you need to make is that rename this script folder to Video-To-Audio(CLI-Ver). Rest of the changes are mentioned below:

**Comment by smriti1313** [comment]
> Hi @smriti1313,
> As there was a confusion of Scripts, the final decision is that both: your and @kshittijagrawal contribution will be considered.
> 
> Now the important change you need to make is that rename this script folder to Video-To-Audio(CLI-Ver). Rest of the changes are mentioned below:

@kaustubhgupta Thank you so much for this. I will make all the necessary changes by afternoon~evening. Thank you☀️

**Comment by smriti1313** [comment]
Hi, I have made the necessary changes. Kindly review it and let me know if it is lacking something or not.
Thank you

**Comment by kaustubhgupta** [comment]
@smriti1313 you pushed the latest Repo changes as commits in this PR which can't be merged. You only need to push your code and not any other changes. To successfully get the changes without merge conflicts, you should have pulled all the changes via upstream. Now kindly remove other files.

**Comment by smriti1313** [comment]
Hi, apologies for what happened. I will send one more PR with proper commits and stuff.
Thank you.

**Pull request closed without merging**